src/main/java/com/rocketden/main/model/Room.java

```diff
@@ -130,6 +132,10 @@ public boolean containsUserWithNickname(String nickname) {
      * than the maximum size of the room
      */
     public boolean isFull() {
-        return users.size() >= maxSize;
+        if (size == RoomService.MAX_SIZE + 1) {
+            return false;
+        }
```

Comment:
nit: Add comment explaining why this is in place, either in the method header or right above the if statement.